skeleton/src/Entity/classBombeiro.php

```diff
@@ -0,0 +1,172 @@
+<?php
+
+require_once 'skeleton/src/Entity/classServico.php';
+
+class Bombeiro {
+    // Atributos
+    private $nome;
+    private $cpf;
+    private $numeroCelular;
+    private $antiguidade = 0;
+    private $carteiraAmbulancia = false;
+    private $pontuacao = 0;
+    private $cidadeOrigem;
+    private $dia = [];
+    private $mes;
+    private $turno; // manhã, tarde, noite, integral(24 horas?)
+
+    // Construtor
+    public function __construct($nome, $cpf, $numeroCelular, $antiguidade, $carteiraAmbulancia, $cidadeOrigem){
+        $this->setNome($nome);
+        $this->setCpf($cpf);
+        $this->setNumeroCelular($numeroCelular);
+        $this->setAntiguidade($antiguidade);
+        $this->setCarteiraAmbulancia($carteiraAmbulancia);
+        $this->setCidadeOrigem($cidadeOrigem);
+
+        //
+
+        switch ($antiguidade) {
+            case $antiguidade < 2:
+                $this->setPontuacao(5);
+                break;
+            case $antiguidade > 2 && $antiguidade < 5:
+                $this->setPontuacao(10);
+                break;
+            case $antiguidade >= 5:
+                $this->setPontuacao(15);
+                break;
+            case $antiguidade < 10:
+                $this->setPontuacao(20);
+                break;
+        }
+
+        if ($nome == "Querubin") {
+            $this->setPontuacao(1000); // Exemplo de pontuação especial
+        }
+
+        switch ($cidadeOrigem) {
+            case "Videira":
+                $this->setPontuacao($this->getPontuacao() + 20);
```

Comment:
Não precisamos ficar get and set dentro da função. Estes métodos são mais úteis para quando estamos manipulando um objeto de fora. Aqui podemos ler e me modificar o atributo com `$this->pontuacao = xxx`